js/about_start.js

```diff
@@ -20,7 +20,7 @@ $('.environment').text(states.join(' - '));
 
 // Install the 'dismiss with escape key' handler
 $(document).on('keydown', e => {
-  if (e.keyCode === 27) {
+  if (e.keyCode === /* escape key */27) {
```

Comment:
Maybe instead of a comment assign this to a variable instead? Just so that if it get's updated you only have to update a variable.

```suggestion
  const escapeKey = 27;
  
  if (e.keyCode === escapeKey) {
```

Reply:
This is how it's done everywhere in the APP.  I'm not refactoring every component :)